ras_party/controllers/party_controller.py

```diff
@@ -487,7 +485,7 @@ def put_email_verification(token, session):
         raise RasError(msg, 404)
 
     try:
-        r = session.query(Respondent).filter(Respondent.email_address == email_address).one()
+        r = _query_respondent_by_email(email_address, session)
```

Comment:
sql query using .first() does not raise NoResultFound, it will return a None. Will have to replace try except with 'if not r:  raise RasError...' statement.